examples/raytracing-rahit/shader.rahit

```diff
@@ -0,0 +1,18 @@
+#version 460 core
+#extension GL_EXT_ray_tracing : require
+
+layout(location = 0) in float opacity; // 0.0f = transparent, 1.0f = opaque
+
+void main()
+{
+    float o = opacity;
+    if (o > 1.0f)
+        o = 1.0f;
+    else if (o < 0.0f)
+        o = 0.0f;
+
+    if (o < 1.0f)
+        ignoreIntersectionEXT;
```

Comment:
This example is interesting because ignoreIntersectionEXT and terminateRayEXT don't do anything presently. Maybe we can keep this example for the time being because it provides another shader stage, but it doesn't make sense to keep it independent indefinitely if it needs to run a closest hit shader or what not.